content/en/synthetics/_index.md

```diff
@@ -8,73 +8,49 @@ further_reading:
 - link: "https://www.datadoghq.com/blog/introducing-synthetic-monitoring/"
   tag: "Blog"
   text: "Introducing Datadog Synthetics"
-- link: "/synthetics/api_tests/"
+- link: "/synthetics/troubleshooting/"
   tag: "Documentation"
-  text: "Configure an API Test"
-- link: "/synthetics/browser_tests/"
+  text: "Troubleshooting Synthetics"
+- link: "/synthetics/settings/"
   tag: "Documentation"
-  text: "Configure a Browser Test"
-- link: "/synthetics/identify_synthetics_bots/"
-  tag: "Documentation"
-  text: "Identify Synthetics Bots"
+  text: "Synthetics Settings"
 ---
 
-## Overview
-
-Datadog Synthetics gives you a new layer of visibility on the Datadog platform. By monitoring your applications and API endpoints via simulated user requests and browser rendering, Synthetics helps you ensure uptime, identify regional issues, and track application performance. By unifying Synthetics with your metrics, traces, and logs, Datadog allows you to observe how all your systems are performing as experienced by your users.
-
 {{< img src="synthetics/synthetics_home_page.png" alt="Synthetics home page" >}}
 
-## Search
-
-Advanced search lets you query checks by any combination of checks attributes:
-
-* `title` and `message` - text search
-* `status` - Alert, No Data, Ok
-* `creator`
-* `region`
-* `muted`
-* `notification`
-* `tags`
+Synthetic tests allow you to observe how all your systems are performing as experienced by your users from around the globe -  Datadog tracks the performance of your applications and API endpoints using simulated user requests and browser rendering. Synthetics helps you ensure uptime, identify regional issues, compute error budgets, automate tests, and track application performance. By unifying Synthetics with your metrics, traces, and logs, you can test your production environment end-to-end.
 
-To run a search, construct your query using the checkboxes on the left and/or the search bar along the top. The search bar updates with the equivalent query as you check the boxes. Likewise, when you modify the search bar query (or write one from scratch), the checkboxes update to reflect that change. Query results update in real-time as you edit the query; there's no 'Search' button to click.
+## Getting Started
 
-### Write a query
+### Set up API tests
 
-To search for specific text across all checks, titles, and notification messages, enter your text in the search bar.
+Monitor how your customers from around the world experience your http API endpoints using [Synthetics API tests][1].
 
-Otherwise, use Boolean operators (`AND`, `OR`, and `NOT`) and parentheses to write complex queries using any check field:
+{{< img src="synthetics/api.png" alt="API tests"  style="width:100%;">}}
```

Comment:
That appears too small :'( let's use a screenshot of the context panel only: https://a.cl.ly/geuwqxG9